01_intro/99_hw/game/main.go

```diff
@@ -1,29 +1,176 @@
 package main
 
-/*
-	код писать в этом файле
-	наверняка у вас будут какие-то структуры с методами, глобальные перменные ( тут можно ), функции
-*/
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Player struct {
+	Inventory       map[string]bool
+	WearingBackpack bool
+	CurrentPlace    *Place
+}
+
+type Place struct {
+	Description string
+	Items       map[string]string
+	Exits       map[string]*Place
+	Actions     map[string]func(*Player) string
+}
 
 func main() {
-	/*
-		в этой функции можно ничего не писать
-		но тогда у вас не будет работать через go run main.go
-		очень круто будет сделать построчный ввод команд тут, хотя это и не требуется по заданию
-	*/
+	initGame()
+
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Println("Добро пожаловать в игру! это самый первый вариант игры(тестовый) Введите команду:")
+
```

Comment:
```suggestion
	reader := bufio.NewReader(os.Stdin)
	fmt.Println("Добро пожаловать в игру! это самый первый вариант игры Введите команду:")

```
Можно убрать тестовые строки